pages/item-new/_.vue

```diff
@@ -17,7 +17,7 @@
     >
       <b-container
         fluid
-        class="bg-white mb-3"
+        class="bg-white mb-3 px-0"
       >
         <!-- TODO: add swiper here
```

Comment:
Could you remove the todos as it is now done?

Reply:
ok